process/phases.md

```diff
@@ -0,0 +1,150 @@
+# WebAssembly W3C Process
+
+This file describes how feature proposals will progress through
+the standardization process.
+
+In general, the process moves forward through a series of numbered phases.
+However, if issues are uncovered or consensus devolves,
+proposals should back up to the appropriate prior step.
+ 
+## 0. Feature Proposal [CG]
+
+Entry requirements:
+  * A CG member has an idea.
+ 
+During this phase a new repository forking the spec repo is created (CG members
+can request this). PRs + Issues iterate on the design of the feature. If human
+readable design repo style .md files are useful, these are added to describe
+the feature as it's being designed. A tracking issue is created in the design
+repo.
+ 
+If relevant to demonstrate the viability of a feature, prototype
+implementations of the feature are implemented in interested engines (possibly
+on a branch).
+The CG will attempt to reach broad consensus.
+ 
+## 1. Proposed Spec Text Available [CG/WG]
+
+Entry requirements:
+   * Full proposed english spec text available in a forked repo around which a
+     reasonably high level of consensus exists.
+   * *Updates to the formal notation, test suite,
+      and reference interpreter are NOT yet required*.
+ 
+During this phase, one or more implementations proceed on prototyping the
+feature to the point that a comprehensive set of tests can be added. These
+tests need not pass the reference interpreter at this point, but should pass on
+some implementation.
+ 
+## 2. Implementation Phase [CG/WG]
+
+Entry requirements:
+   * Test suite has been updated to cover the feature in its forked repo.
+   * The test suite should run against some implementation, though it need not be
+     the reference interpreter.
+   * *Formal notation in the spec need not be updated.*
+ 
+During this phase, the following proceeds in parallel:
+Web VMs (browsers) implement the feature.
+The forked repo is updated to include revisions to the formalisation.
+The forked repo spec is updated to include implementation of the feature in the
+reference interpreter (OCaml).
+The feature is implemented in toolchains.
+ 
+## 3. Standardize the Feature [WG]
+
+Entry requirements:
+   * Two or more Web VMs implement the feature.
```

Comment:
I think we also need some signal from others that there's interest in implementing. I think it's another one of those "the Chair decides what looks like consensus, but if people consistently feel like the Chair ignores them then things will break down".